package.json

```
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "@gimenete/type-writer": "^0.1.5",
+    "@octokit/tsconfig": "^1.0.0",
```

Update the version once it's out, then this is good to go

```suggestion
    "@octokit/tsconfig": "^1.0.1",
```

---

you will have to update the lock file, too